connectors/zapier/connector.py

```diff
@@ -76,6 +76,15 @@ def hand_off_to_zapier(entity_name, message):
                     logging.info(has_attribution_ratingv2)
                 else:
                     hand_off_to_zapier(entity_name, message)
+            elif 'completed_at' in message['payload'] and message['payload']['product_id'].lower() == 'ratingv3':
+                if 'marketing_attribution' in message['payload']:
+                    logging.info(has_attribution_ratingv2)
+                elif 'person_phone' in message['payload'] and 'appointment_start_date_and_time' in message['payload']:
+                    # second zapier hook. the initial webhook is sent after completed_at=true and email given
+                    message['payload']['product_id'] = 'ratingv3_with_phone'
```

Comment:
manually manipulate the product_id, since `hand_off_zapier` highly relies on product id and from FE we get only one.
In this way also we can differentiate the zapier hooks.